lib/phlex/tag.rb

```diff
@@ -7,13 +7,10 @@ class Tag
     UNDERSCORE = "_"
     NAMESPACE_DELINEATOR = "::"
 
-    class << self
-      def tag_name
-        @tag_name ||= name.split(NAMESPACE_DELINEATOR).last.downcase
-      end
-    end
+    attr_reader :name
 
-    def initialize(**attributes)
+    def initialize(name:, **attributes)
```

Comment:
Let's use a positional argument for the name so the `name` keyword argument can be a tag attribute, e.g. `<div name="something">`.

Reply:
Nice catch! Just added the change in the last commit.